test/libsolidity/syntaxTests/modifiers/cross_contract_super.sol

```diff
@@ -0,0 +1,9 @@
+contract C {
+    modifier m() { _; }
+}
+contract D is C {
+    function f() super.m public {
+    }
+}
+// ----
+// DeclarationError 7920: (74-81): Identifier not found or not unique.
```

Comment:
It's also not possible with modifiers (though I found a different problem: #11468).